components/external_server/src/utils/functions.py

```diff
@@ -19,6 +19,7 @@
 # and Dlint complains 'insecure use of XML modules, prefer "defusedxml"'
 # but we give autolink_html clean html, so ignore the warning:
 from lxml.html.clean import autolink_html, clean_html  # noqa: DUO107, # nosec # pylint: disable=no-name-in-module
+import lxml  # noqa: DUO107, # nosec
```

Comment:
Is it possible to only import `fromstring` & `tostring` instead of the entire library ?

Reply:
Perhaps we should also look at using `BeautifulSoup` instead of `lxml`, although I guess we can't drop `lxml` entirely due to other requirements :(

Reply:
Or bleach for cleaning HTML.